### PR TITLE
update the deployer-related READMEs

### PR DESCRIPTION
**Pull request by escattone**

### CODE REVIEW

File: deployer/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 The Yari deployer does two things. First, it's used to upload document
 redirects, pre-built document pages, static files (e.g. JS, CSS, and
-image files), and sitemap files into an existing AWS S3 bucket.
-Since we serve MDN from a S3 bucket via a CloudFront CDN, this is the
-way we upload a new version of the site.
+image files), and sitemap files into an existing AWS S3 bucket. Since
+we serve MDN document pages from a S3 bucket via a CloudFront CDN,
```
Comment (peterbe) [inline deployer/README.md:6]: Interestingly(?), I use Grammarly and they recommend using `an` when the next word starts with an `s`.

File: deployer/README.md [hunk 2]
```diff
@@ -28,19 +28,23 @@ with regards to configuring AWS access credentials.
 ## Uploads
 
 The `poetry run deployer upload` command uploads files as well as redirects
```
Comment (peterbe) [inline deployer/README.md:30]:
```suggestion
The `poetry run deployer upload DIRECTORY` command uploads files as well as redirects
```

File: deployer/README.md [hunk 3]
```diff
@@ -28,19 +28,23 @@ with regards to configuring AWS access credentials.
...
-into an existing S3 bucket. As input, it takes a directory which contains the
-files that should be uploaded, but it also needs to know where to find
-any redirects that should be uploaded. By default it searches for redirects
-within the content directories specified by `--content-root` (or
-`CONTENT_ROOT`) and `--content-translated-root` (or `CONTENT_TRANSLATED_ROOT`).
+into an existing S3 bucket. Currently, we have three main S3 buckets that we
+upload into: `mdn-content-dev` (for variations or experimental versions of
+the site), `mdn-content-stage`, and `mdn-content-prod`.
+
+As input, the `poetry run deployer upload` command takes a directory which
```
Comment (peterbe) [inline deployer/README.md:35]:
```suggestion
As input, the `upload` command takes a directory which
```
so you don't need to repeat the whole command again. Especially if we need to change it in two places some day.

**Review by peterbe — APPROVED** [review]
only nits

File: deployer/README.md (hunk 1 above)
Comment (escattone) [reply deployer/README.md:6]: I totally agree with Grammarly, and I sometimes I remember to do that, but I'm not consistent! Thanks.

**Pull request merged**